test/config/offlinetargettest.go

```diff
@@ -51,19 +51,19 @@ func (s *TestSuite) TestOfflineDevice(t *testing.T) {
 		},
 	}
 	extensions := []*gnmi_ext.Extension{{Ext: &extNameDeviceType}, {Ext: &extNameDeviceVersion}}
-	devicePath := gnmi.GetDevicePathWithValue(offlineDeviceName, modPath, modValue, proto.StringVal)
+	devicePath := gnmi.GetTargetPathWithValue(offlineDeviceName, modPath, modValue, proto.StringVal)
 	networkChangeID := gnmi.SetGNMIValueOrFail(t, gnmiClient, devicePath, gnmi.NoPaths, extensions)
 
 	// Bring device online
 	simulator := gnmi.CreateSimulatorWithName(t, offlineDeviceName)
 	defer gnmi.DeleteSimulator(t, simulator)
 
 	// Wait for config to connect to the device
-	gnmi.WaitForDeviceAvailable(t, device.ID(simulator.Name()), time.Minute)
+	gnmi.WaitForTargetAvailable(t, device.ID(simulator.Name()), time.Minute)
```

Comment:
there is no device ID anymore. Under onos-api/go/v2 you will be find these types that you can use for the tests: https://github.com/onosproject/onos-api/blob/master/go/onos/config/v2/types.go#L53

Reply:
These are still using the "device shim" and will be changed when that is eliminated.

Reply:
I think you can go ahead and update it  unless you want to do it separately which is fine.